renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": ["config:base"],
+  "semanticPrefix": "build(all):",
+  "lockFileMaintenance": {"enabled": true},
+  "packageRules": [{"packagePatterns": ["^@types/"], "automerge": true}]
```

Comment:
I think [this](https://renovatebot.com/docs/configuration-options/#automerge) applies here:

> Warning: GitHub currently has a bug where automerge won't work if a GitHub Organization has protected their master branch, and there is no way to configure around this. Hence, automerging will try and fail in such situations.